### DISCUSSION TITLE
Wi-Fi is not working properly

### DISCUSSION THREAD

**Discussion by sasha9797** [daliansky/XiaoMi-Pro#729] [Q&A]
Wi-Fi does not work correctly, it is also possible to connect to the access point for a long time, disconnect from the access point during operation, what can be done about this?

**Comment by stevezz3** [comment]
You can go to https://github.com/OpenIntelWireless/itlwm/issues for help on Intel Wi-Fi related problems. They recently released v2.2.0 stable version, which worth a try. Or you can wait our next release to include that Wi-Fi update.
